corehq/form_processor/abstract_models.py

```
@@ -212,7 +212,8 @@ def resolve_case_property(self, property_name):
         case has more than one parent or grandparent.
         """
         result = []
-        self._resolve_case_property(property_name, result)
+        if isinstance(property_name, basestring):
+            self._resolve_case_property(property_name, result)
```

fair point, i updated it to let the exception raise